### DISCUSSION TITLE
Can the "Vector Zonal Stats" function be renamed to something else?

### DISCUSSION THREAD

**Discussion by tm-ardie-orden** [thinkingmachines/geowrangler#56] [Enhancements Requests]
@butchtm @jtmiclat I'm thinking that "Vector Zonal Stats" might be bad for SEO purposes. When I search "Zonal Stats", all the search results show raster-based algorithms. I'm thinking that it might be better to rename it to "Spatial Join (Summary Stats)" or "Spatial Join (Aggregate Stats)" or something.

What do you think?

**Comment by jtmiclat** [comment]
I do agree that Vector Zonal Stats isnt great at the same time would like a name that works symmetrically with rasters. hence the current names `Raster Zonal Stats` and `Vector Zonal Stats`. Some possible names, 
- `Vector Aggregates` and `Raster Aggregates` 
- `Vector Summary Stats` and `Raster Summary Stats`

**Reply by tm-ardie-orden** [reply]
I understand. I personally prefer "Zonal Statistics" for the raster version and "Spatial Join Aggregates" for the vector version.

But I leave the final decision up to the dev team! This is just a nitpick regarding the name.
